src/reanimated2/js-reanimated/index.ts

```diff
@@ -36,6 +36,11 @@ if (shouldBeUseWeb()) {
       "[Reanimated] You can't use `scrollTo` with Chrome Debugger or with web version"
     );
   };
+  global._dispatchCommand = () => {
+    console.warn(
+      "[Reanimated] You can't use `dispatchCommand` or `scrollTo` methods with Chrome Debugger or with web version"
```

Comment:
Shouldn't it be just `dispatchCommand` as the same message for `scrollTo` is defined above?

Reply:
On Paper, `scrollTo` function from Reanimated API uses `_scrollTo` JSI binding.

On Fabric, I've implemented `scrollTo` as a specific case of `dispatchCommand` which uses `_dispatchCommand` JSI binding.

We would like to mention both functions in the warning message in order to avoid confusion why this warning shows up even if user doesn't use `dispatchCommand` function directly.

A better way would be to move this `console.warn` directly to `scrollTo` and `dispatchCommand` functions.